### PR TITLE
MINOR: add GlobalKTable doc to streams.html

### PR DESCRIPTION
**Pull request by dguy**
Update streams.html with GlobalKTable docs

### CODE REVIEW

**Comment by dguy** [comment]
@enothereska @mjsax @guozhangwang @miguno - this is really just a c&p from the existing docs

**Review by miguno — APPROVED** [review]

File: docs/streams.html [hunk 1]
```diff
@@ -378,7 +385,13 @@
...
+        <li><b>KStream-to-GlobalKTable Joins</b> allow you to perform table lookups against a fully replicated changelog stream (<code>GlobalKTable</code>) upon receiving a new record from another record stream (<code>KStream</code>).
+            Joins with a <code>GlobalKTable</code> don't require repartitioning of the input <code>KStream</code> as all partitions of the <code>GlobalKTable</code> are available on every KafkaStreams instance.
+            A <code>KeyValueMapper</code> provided with the join operation is used to lookup the value from the <code>GlobalKTable</code> so non-key joins are possible.
```
Comment (mjsax) [inline docs/streams.html]: That's unclear: The `KeyValueMapper` provided with the join operation is applied to each KStream record to extract the join-key that is used to do the lookup to the GlobalKTable so non-record-key joins are possible.

**Review by mjsax — COMMENTED** [review]
One nitpick. Otherwise LGTM.

File: docs/streams.html (hunk 1 above)
Comment (dguy) [reply docs/streams.html]: suggestion? :-)

Comment (mjsax) [reply docs/streams.html]: That was already a suggestion:

`The <code>KeyValueMapper</code> provided with the join operation is applied to each KStream record to extract the join-key that is used to do the lookup to the GlobalKTable so non-record-key joins are possible.`

Comment (dguy) [reply docs/streams.html]: oh, yeah! sorry

File: docs/streams.html [hunk 2]
```diff
@@ -497,25 +497,31 @@
...
 
-        If these records a KStream and the stream processing application were to sum the values it would return <code>4</code>. If these records were a KTable, the return would be <code>3</code>, since the last record would be considered as an update.
+        If these records a KStream and the stream processing application were to sum the values it would return <code>4</code>. If these records were a KTable or GlobalKTable, the return would be <code>3</code>, since the last record would be considered as an update.
 
         <h4><a id="streams_dsl_source" href="#streams_dsl_source">Create Source Streams from Kafka</a></h4>
 
         <p>
-        Either a <b>record stream</b> (defined as <code>KStream</code>) or a <b>changelog stream</b> (defined as <code>KTable</code>)
-        can be created as a source stream from one or more Kafka topics (for <code>KTable</code> you can only create the source stream
+        Either a <b>record stream</b> (defined as <code>KStream</code>), <b>changelog stream</b> (defined as <code>KTable</code>), or a <b>changelog stream</b> (defined as <code>GlobalKTable</code>)
```
Comment (guozhangwang) [inline docs/streams.html]: nit: or a <b>changelog stream</b> (defined as <code>KTable</code> or <code>GlobalKTable</code>) ..

**Review by guozhangwang — COMMENTED** [review]
One nitpick, otherwise LGTM.

**Review by guozhangwang — APPROVED** [review]
Merged to trunk and 0.10.2.

**Pull request closed without merging**